config/honeybadger.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
to avoid having multiple files for configuration, can we move the entire configuration inside the initializer?

It's nice to see the complete configuration in one glance, see https://docs.honeybadger.io/lib/ruby/gem-reference/configuration.html#configuration-options for the keys in Ruby